dvc/command/checkout.py

```diff
@@ -1,18 +1,91 @@
 import argparse
+import logging
+
+import colorama
 
 from dvc.command.base import append_doc_link
 from dvc.command.base import CmdBase
+from dvc.exceptions import CheckoutError
+
+logger = logging.getLogger(__name__)
+
+
+def _humanize_wordseries(words=None):
+    words = list(words or [])
+    num_words = len(words)
+    if num_words == 0:
+        return ""
+    if num_words == 1:
+        return words[0]
+
+    return "{part1}{comma} and {part2}".format(
+        part1=", ".join(words[:-1]),
+        comma="," if num_words > 2 else "",
+        part2=words[-1],
+    )
+
+
+def _log_summary(stats, with_failure=True):
+    template = [
+        "{added} added",
+        "{deleted} deleted",
+        "{modified} modified",
+    ]
+    if with_failure:
+        template.append("{failed} failed")
+
+    summary = {stat: len(num) for stat, num in stats.items()}
+    logger.info(_humanize_wordseries(template).format_map(summary))
+
+
+def _log_changes(stats, with_failure=False):
+    colors = [
+        ("added", colorama.Fore.GREEN),
```

Comment:
does it makes to utilize LS_COLORS as well? may be check how does git do this with diffs?

Reply:
I've checked git, and it works based on `diff.color`/`color.diff` config option
[diff.c](https://github.com/git/git/blob/bfdd66e72fffd18235757bedbf355fd4176d6019/diff.c#L57)
[diff.h](https://github.com/git/git/blob/b02fd2accad4d48078671adf38fe5b5976d77304/diff.h#L389)
[diff.color.](https://github.com/git/git/blob/bfdd66e72fffd18235757bedbf355fd4176d6019/diff.c#L435)
```
$ git config --local color.diff.new yellow
$ git log -p
```
https://stackoverflow.com/a/26941235/484582
<hr/>

but gnu diffutils with [this commit](http://git.savannah.gnu.org/cgit/diffutils.git/commit/?id=04f6d57177fb6c9e7bbcf716c521bc323936a9a5) started to use theme with the same notation as ls does
But it isn't propagated from the env variable
```
defaults to @samp{rs=0:hd=1:ad=32:de=31:ln=36} for red deleted lines,
green added lines, cyan line numbers, bold header
```
```
$ echo 1 > file1
$ echo 2 > file2
$ diff file1 file2 --color --palette="ad=01;35:de=01;36"
...
```

Reply:
The current output is very similar to `dvc diff`.

> does it makes to utilize LS_COLORS as well?

you mean, colorizing pathnames?

Another possible output format to use is of `git checkout`:
```sh
$ git checkout branch7
M	.gitignore
A	data2/data
Switched to branch 'branch7'
```

Reply:
I meant more of a reusing some colorization/output style of Git (diff first came to my mind with red for removed, green for added). Idea to mimic `git checkout` is also good.

Instead of hardcoding color codes utilize some environment (in a way Gut did his reserach)

Reply:
We could color it after #3351 is merged. It'd be better to go with however `ls` works (and, not have too many color formats).

`git checkout` style is easily greppable, whereas `dvc diff` style output is not. 
```diff
$ dvc checkout
+ M data
! D foo
+ A models
- F something
# ignore the colors and symbols
```

We could make the pathnames follow `LS_COLORS`, whereas keep *{M,A,D,F}* as {yellow, green, orange, and red}* (or, something even better). What do you think, @shcheklein?

Reply:
But, I assume `dvc diff` can be easily understood.

Reply:
I like it! Both idea to use git checkout format and LS_COLORS.

Reply:
Let's discuss on `LS_COLORS` separately. I have made the output similar to `git checkout`.